certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -0,0 +1,225 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "CyberProof Actions",
+    "description": "This connector will enable you to execute CyberProof Actions.",
```

Comment:
@DavidMeu Your connector description is short and vague that everyone can not understand about your product or service. Please follow the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and have a detailed description for your connector that should describe concisely the main purpose and value offered by your connector.

Reply:
@sriyen-msft 
Updated